BitwardenShared/UI/Vault/Vault/VaultItemSelection/VaultItemSelectionView.swift

```diff
@@ -104,25 +104,59 @@ private struct VaultItemSelectionSearchableView: View {
                 get: \.toast,
                 send: VaultItemSelectionAction.toastShown
             ))
+            .background(Color(asset: Asset.Colors.backgroundSecondary).ignoresSafeArea())
     }
 
     // MARK: Private Views
 
     /// The content displayed in the view.
     @ViewBuilder
     private func contentView() -> some View {
-        VStack(spacing: 0) {
-            if isSearching {
-                searchContentView()
-            } else {
-                if store.state.vaultListSections.isEmpty {
-                    // TODO: BIT-2350 Add empty view
-                } else {
-                    matchingItemsView()
+        if isSearching {
+            searchContentView()
+        } else {
+            if store.state.vaultListSections.isEmpty {
+                GeometryReader { reader in
+                    VStack(spacing: 24) {
+                        Asset.Images.openSource.swiftUIImage
+                            .resizable()
+                            .frame(width: 100, height: 100)
+                            .padding(.bottom, 8)
```

Comment:
🤔 Can the image be renamed? Is there a relationship between `openSource` and an image for empty things? Specifically because when reading the code one should know what the image represents. Maybe something like `emptyListPlaceholder`, what do you think?

Reply:
From what I've seen, this image only applies to this specific screen. So I guess we could name it specific for this screen, but I've found that there's less confusion if we match assets to what they are named in the designs. And I think it's more obvious with using the empty view.

Reply:
Makes sense to leave it as is, thanks! 🎉